SocialFilter/sections/specific.txt

```diff
@@ -2217,7 +2217,7 @@ family-style.net##.mvp-post-soc-fb
 dnepr.info##.soc-tabs
 vinnytsianews.com##.row > aside.widget-area > aside.widget_fbw_id
 vinnytsianews.com##.row > aside.widget-area > aside.widget-instagram
-vinnytsianews.com##.share-btns-main
+whatsnewinpublishing.com,vinnytsianews.com##.share-btns-main
```

Comment:
I don't see this element. Found missed button and added `whatsnewinpublishing.com##.themezee-social-sharing`

Reply:
@Alex-302 
![image](https://user-images.githubusercontent.com/76908410/119593089-313e8d00-be0c-11eb-801c-faf8f66eb700.png)